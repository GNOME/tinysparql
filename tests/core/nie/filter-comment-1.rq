SELECT ?comment
WHERE { ?x nie:comment ?comment }
ORDER BY ?comment
