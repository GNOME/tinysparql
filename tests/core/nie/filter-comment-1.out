"stringly data for nie:comment"
